### DISCUSSION TITLE
Why do all createSSGHelpers examples use ISR?

### DISCUSSION THREAD

**Discussion by penrodlol** [trpc/trpc#2579] [Q&A] [answered]
I assume I'm missing some fundamentals somewhere but why do all examples of `createSSGHelpers` use ISR? And why is the revalidation set to 1?

I have a few pages where data is static and don't need to fetch every X amount of time. I'm hesitant to opt the page into ISR if I don't need to. But perhaps its necessary for the function to work properly... Any feedback would be appreciated here. Thanks!

![Screenshot from 2022-08-28 15-44-50](https://user-images.githubusercontent.com/53282218/187091761-6bd521e1-7ef2-46c6-b7c1-2820c77df1b4.png)

**Comment by KATT** [comment, marked as answer]
No reason more than for the sakes of docs, what you do is up to you!

**Reply by penrodlol** [reply]
@KATT  Sounds good, thanks!
